src/components/NavBar/index.js

```diff
@@ -21,21 +24,61 @@ class NavBar extends Component {
     });
   }
 
+  handleResize = () => {
+    this.setState({ windowWidth: window.innerWidth });
+  }
+
+  componentDidMount() {
+    window.addEventListener("resize", this.handleResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.handleResize);
+  }
+
   render() {
     const { selectedKeys } = this.state;
+    const { windowWidth } = this.state;
+
     return (
-      <Header>
-        <Menu
-          theme="dark"
-          mode="horizontal"
-          selectedKeys={selectedKeys}
-          style={{ lineHeight: '64px' }}
-        >
-          <Menu.Item key="/" onClick={() => this.setSelectedKeys(['/'])}><Link to="/">Home</Link></Menu.Item>
-          <Menu.Item key="/upload-worklog-report" onClick={() => this.setSelectedKeys(['/upload-worklog-report'])}><Link to="/upload-worklog-report">Upload Report</Link></Menu.Item>
-          <Menu.Item key="/view-payroll-report" onClick={() => this.setSelectedKeys(['/view-payroll-report'])}><Link to="/view-payroll-report">View Payroll Report</Link></Menu.Item>
-        </Menu>
-      </Header>
+      <>
+        { windowWidth > mobileWidthBreak ?
+          <Header>
+            <Menu
```

Comment:
Actually we can declare the `<Menu>` and its content as `const` above and use it in Header and Sidebar.

Thus we follow DRY principle https://en.wikipedia.org/wiki/Don%27t_repeat_yourself